module-2/backend/week_two.md

```diff
@@ -27,12 +34,24 @@ end
 Now how would you find the owner of the team with an id of 4?
 
 3. What do they allow you to do?
+
+*?????*
 7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+  student class => has_many :teachers
+  teacher class => has_many :students
+
 8. Define foreign key, primary key, and schema.
+  foreign key = a foreign key references a row at another table, it will have that tables name followed by 'id'
+  primary key = it's a unique identifier for a table's row/object
+  schema = it's the blueprint of the tables
+  
 9. Describe the relationship between a foreign key on one table and a primary key on another table.
+  The foreign key corresponds to the primary key from another table. 
+  
 10. What are the parts of an HTTP response?
+  Start line, header and body. 
```

Comment:
Think that first part maybe you were thinking of status? An HTTP response has headers and a body and the first header line is a status code (e.g. 404 - not found, 200 - ok, 503 - server error, etc).